compiler+runtime/src/cpp/jank/runtime/module/loader.cpp

```diff
@@ -298,6 +298,45 @@ namespace jank::runtime::module
     }
   }
 
+  static void
+  register_module_path(native_unordered_map<jtl::immutable_string, loader::entry> &entries,
+                       jtl::immutable_string const &paths,
+                       bool skip_jar = false)
+  {
+    usize start{};
+    usize i{ paths.find(loader::module_separator, start) };
+
+    /* Looks like it's either an empty path list or there's only entry. */
+    if(i == jtl::immutable_string::npos)
+    {
+      if(!skip_jar || !paths.ends_with(".jar"))
+      {
+        register_path(entries, paths);
+      }
+    }
+    else
+    {
+      while(i != jtl::immutable_string::npos)
+      {
+        auto const path(paths.substr(start, i - start));
+
+        if(!skip_jar || !path.ends_with(".jar"))
```

Comment:
This will prevent new jars from being added to the path, right? That prevents the use case of adding a new dependency at runtime.